install/updateVersion.sh

```diff
@@ -64,6 +69,10 @@ while getopts :i:n:p:x:c:a:h:r:P:d:D:m: arg; do
     P) PILOT_DEBIAN_URL="${OPTARG}";;
     d) DEST_DIR="${OPTARG}";;
     D) PROXY_DEBUG="${OPTARG}";;
+    s) STATSD_HUB_TAG="${OPTARG}";;
```

Comment:
As I read things it seems like the original/default hub/tag for these 4 needs to be added to istio.VERSION as part of this change.

Reply:
Right. Will add them here.